build/stf-run-ci/tasks/preflight_checks.yml

```diff
@@ -1,15 +1,26 @@
 ---
 # Try for 10 minutes to get an output
-- name: "Wait for up to 10 minutes for Service Telemetry Operator to be Succeeded"
-  ansible.builtin.shell: |
-    oc get csv -n "{{ namespace }}" | grep service-telemetry-operator | grep Succeeded
-  register: output
-  retries: 60
-  delay: 10
-  until: output.stdout | length != 0
-  ignore_errors: true
+- block:
+    - name: "Wait for up to 30 minutes for Service Telemetry Operator to be Succeeded"
+      ansible.builtin.shell: |
+        oc get csv -n "{{ namespace }}" | grep service-telemetry-operator | grep Succeeded
+      register: output
+      retries: 60
+      delay: 30
```

Comment:
I think there might be some network delays in the CI environment, I wanted to give it more time to see if it would succeed, so I could determine if it's an external issue or an issue with the CSV.

Reply:
The local_build-index_deploy job is working now, I can update this to be 10 minutes again